hasher-matcher-actioner/.devcontainer/zshrc

```diff
@@ -12,4 +12,8 @@ setopt INC_APPEND_HISTORY
 # Make path respect local binaries from pip
 export PATH=$PATH:~/.local/bin
 
+# Add directory containing hmalib to PYTHONPATH. This allows vscode
+# testdiscovery to work
+export PYTHONPATH=$PYTHONPATH:/workspaces/ThreatExchange/hasher-matcher-actioner
```

Comment:
Ignorable (for now) could you either add this to the Dockerfile or create a `.profile` for this. I don't use the default zshrc.
Ref: https://github.com/facebook/ThreatExchange/blob/master/hasher-matcher-actioner/.devcontainer/Dockerfile#L130-L132